index.js

```diff
@@ -197,20 +198,28 @@ async function underPressure (fastify, opts) {
   }
 
   async function onStatus (req, reply) {
+    const response = { status: 'ok' }
```

Comment:
let's give this variable a more sensible name. this is okResponse or something like that. I don't really mind the name, but `response` is a little generic